packages/engine/lib/stateful/src/context/batch.rs

```diff
@@ -19,24 +19,21 @@ use rayon::iter::{
     IndexedParallelIterator, IntoParallelIterator, IntoParallelRefIterator, ParallelIterator,
 };
 
-use crate::{
-    datastore::error::{Error, Result},
-    proto::ExperimentId,
-    simulation::package::context::ContextColumn,
-};
+use crate::{context::ContextColumn, Error, Result};
 
 // If required data size is 3 times less than current data size then shmem size will be readjusted.
 const UPPER_BOUND_DATA_SIZE_MULTIPLIER: usize = 3;
 
-pub type AgentIndex = (u32, u32);
-pub type MessageIndex = (u32, u32, u32);
-
-/// Contains the information required to build the `context` object accessible in the language
+/// Contains the information required to build the [`Context`] object accessible in the language
```

Comment:
Is the `Context` **object** actually accessible in the language runners though? I think the link _might_ be a little misleading.

Reply:
I agree, we should be more careful when naming and linking things. the [`Context`] object is **not** accessible **in** the runners but accessed **by** the runners to provide the context, which is made available.